configs/webpack.dev.js

```diff
@@ -11,4 +11,10 @@ const previewConfig = {
     devtool: "inline-source-map"
 };
 
-module.exports = [merge(baseConfig[0], customConfig), merge(baseConfig[1], previewConfig)];
+const customConfigurations = [merge(baseConfig[0], customConfig), merge(baseConfig[1], previewConfig)];
+
+if (baseConfig.length === 3) {
```

Comment:
// Editor config which is responsible for preview mode ...? should be also mixed in if it exists

Reply:
Done